neurom/io/utils.py

```diff
@@ -198,7 +198,7 @@ def load_neurons(neurons,
                 L.info('Ignoring exception "%s" for file %s',
                        e, f.name)
                 continue
-            raise
+            raise NeuroMError(e)
```

Comment:
up to you but you can add the `from e` which will make the backtrace a bit more explicit

Reply:
done